src/swap/types.ts

```diff
@@ -67,18 +97,48 @@ export type SwapError = {
   error: string; // The error message
 };
 
+/**
+ * Note: exported as public Type
+ */
+export type SwapParams = GetQuoteParams | GetSwapParams;
```

Comment:
I think joinging them will might confuse folks, because now `fromAddress: Address;` which is in Swap, it's not in Quote, but we allow it to have it now when use it in `getQuote`.

Reply:
Should we just keep the two type seprated?

Reply:
Oh ok so you have `getParamsForToken` that depends on both, interesting.